main.tf

```diff
@@ -302,6 +302,33 @@ resource "kubernetes_limit_range_v1" "this" {
   }
 }
 
+################################################################################
+# K8s Admin ClusterRole Binding
+################################################################################
+
+resource "kubernetes_cluster_role_binding_v1" "admin" {
+  count = var.create_cluster_role && var.enable_admin ? 1 : 0
+
+  metadata {
+    name        = coalesce(var.role_name, var.name)
```

Comment:
I don't think we want to do this - lets just have users be explicit in the role name

Reply:
You mean create a new var for the admin_role, or use the existing `var.role_name`?